src/utils/private.ts

```diff
@@ -47,49 +49,77 @@ export const getConfigSuperset = async (context: Context, type: "org" | "repo",
   }
 };
 
-export interface WideLabel {
-  name: string;
-}
+const WideLabelSchema = Type.Object(
+  {
+    name: Type.String(),
+  },
+  { additionalProperties: false }
+);
+
+export type WideLabel = Static<typeof WideLabelSchema>;
+
+const CommandObjSchema = Type.Object(
+  {
+    name: Type.String(),
+    enabled: Type.Boolean(),
+  },
+  { additionalProperties: false }
+);
+
+export type CommandObj = Static<typeof CommandObjSchema>;
+
+const IncentivesSchema = Type.Object(
+  {
+    comment: Type.Object(
+      {
+        elements: Type.Record(Type.String(), Type.Number()),
+        totals: Type.Object(
+          {
+            word: Type.Number(),
+          },
+          { additionalProperties: false }
+        ),
+      },
+      { additionalProperties: false }
+    ),
+  },
+  { additionalProperties: false }
+);
+
+export type Incentives = Static<typeof IncentivesSchema>;
+
+export const WideConfigSchema = Type.Object(
+  {
+    "evm-network-id": Type.Optional(Type.Number()),
+    "price-multiplier": Type.Optional(Type.Number()),
+    "issue-creator-multiplier": Type.Number(),
+    "time-labels": Type.Optional(Type.Array(WideLabelSchema)),
+    "priority-labels": Type.Optional(Type.Array(WideLabelSchema)),
+    "payment-permit-max-price": Type.Optional(Type.Number()),
+    "command-settings": Type.Optional(Type.Array(CommandObjSchema)),
+    "promotion-comment": Type.Optional(Type.String()),
+    "disable-analytics": Type.Optional(Type.Boolean()),
+    "comment-incentives": Type.Optional(Type.Boolean()),
+    "assistive-pricing": Type.Optional(Type.Boolean()),
+    "max-concurrent-assigns": Type.Optional(Type.Number()),
+    incentives: Type.Optional(IncentivesSchema),
+    "default-labels": Type.Optional(Type.Array(Type.String())),
+    "register-wallet-with-verification": Type.Optional(Type.Boolean()),
+  },
+  {
+    additionalProperties: false,
+  }
+);
```

Comment:
Looks good to replace `interfaces` with `schemas` and i guess it would be better for you to move the schemas to `src/types`.